build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.1.0'
+        classpath 'com.android.tools.build:gradle:2.0.0'
```

Comment:
Probably gradle plugin updates should be done in a separate PR ( @blundell  would agree with this :trollface: ), but if we update it, better update it to the latest stable version